File: C/code/sum_two_integers.c
# include <stdio.h>
int main () {
	int num1;
	int num2;
	printf("User, enter first number: ");
	scanf("%d", &num1);
	printf("User, enter second number: ");
	scanf("%d", &num2);
	printf("%d+%d=%d\n",num1, num2, (num1+num2));
}

File: GO/code/sum_two_integers.go
package main
import "fmt"

func main() {
	var num1 int
	var num2 int
	fmt.Println("User, enter first number: ")
	fmt.Scan(&num1)
	fmt.Println("User, enter second number: ")
	fmt.Scan(&num2)
	fmt.Print("The sum of ", num1)
	fmt.Print(" and of ", num2)
	fmt.Println(" is:", (num1+num2))
}
